_authors/osci_2023/kshitijalwadhi.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Kshitij Alwadhi
+short_name: Kshitij
+linkedin: kshitijalwadhi
+github: kshitijalwadhi
+---
+
+**<Kshitij Alwadhi>** <First year MSCS grad at Purdue University>
```

Comment:
Please remove the `<>` brackets:

```suggestion
**Kshitij Alwadhi** First year MSCS grad at Purdue University
```

Reply:
Thanks.
Addressed in the following commit.